### ISSUE TITLE
libGDX Project now in develop branch

### ISSUE THREAD

**Issue by matys18** [matys18/Inda-project#13]
@Jarlius if you pull from the repo and go into the develop branch, you should be able to import the libGDX project into eclipse by doing File -> Import -> Gradle -> Gradle Project

Confirm that it's working and then we can merge :+1:

**Comment by matys18** [comment]
There is a tutorial here https://www.youtube.com/watch?v=IBsvuT7MzpY

**Comment by Jarlius** [comment]
It's working now, though the repository got split into three Projects in the project explorer - Inda-project, placeholder-core and placeholder-desktop (not sure what implications it will have, all these plugins make it feel rather messy).

**Comment by matys18** [comment]
libGDX is cross-platform, so the core project is where the actual code goes, and the desktop project is simply a launcher that launches stuff from the core project. 

This is not relevant to what we're doing, but if we wanted our game to run on Desktop, Android and iOS, there would be a project for all three of them that would configure our game for each specific platform.

TLDR: Code goes in the core project, and you run it from the Desktop project
